default/stringtables/en.txt

```diff
@@ -5372,7 +5378,7 @@ There are four sub windows:  [[DESIGN_WND_STARTS]], [[DESIGN_WND_PART_PALETTE_TI
 
 [[DESIGN_WND_FINISHED_DESIGNS]] designs appear in the same order as on the Production Window.  The right click context menu provides several options.  Obsolete/unobsolete designs with "[[DESIGN_WND_OBSOLETE_DESIGN]]".  Ctrl click also obsoletes/unobsoletes designs.  Delete designs with "[[DESIGN_WND_DELETE_DESIGN]]".  Deleted designs can't be recovered. Ships already queued for production will continue to be built with the old obsolete/deleted design.  Rename designs with "[[DESIGN_RENAME]]".  Save designs to disk for future games with "[[DESIGN_SAVE]]".  Toggle the "[[DESIGN_WND_ADD_ALL_DEFAULT_START]]" option to use/not use the default designs in new games.
 
-Use the [[DESIGN_WND_SAVED_DESIGNS]] designs right click context menu to copy a single or all saved designs to the [[DESIGN_WND_FINISHED_DESIGNS]] designs. Ctrl click also copies a single design to the [[DESIGN_WND_FINISHED_DESIGNS]] designs.  Permanently remove a saved design with "[[DESIGN_WND_DELETE_SAVED]]".  Check "[[OPTIONS_ADD_SAVED_DESIGNS]]" to automaitcally import all the [[DESIGN_WND_SAVED_DESIGNS]] designs at the start of a new game.
+Use the [[DESIGN_WND_SAVED_DESIGNS]] designs right click context menu to copy a single or all saved designs to the [[DESIGN_WND_FINISHED_DESIGNS]] designs. Ctrl click also copies a single design to the [[DESIGN_WND_FINISHED_DESIGNS]] designs.  Permanently remove a saved design with "[[DESIGN_WND_DELETE_SAVED]]".  Check "[[OPTIONS_ADD_SAVED_DESIGNS]]" to automatically import all the [[DESIGN_WND_SAVED_DESIGNS]] designs at the start of a new game.
```

Comment:
also here... Most other test seemingly doesn't use them. Would be good to standardize (probably on single spaces).